tests/test_order.py

```diff
@@ -2,21 +2,21 @@
 """pyetrade authorization unit tests

```

Comment:
the test for `test_place_equity_order` the xml assertion will need to be changed to match the `xmltodict` return.
https://github.com/jessecooper/pyetrade/actions/runs/4473576965/jobs/7872705243#step:5:114

example xml tests from accounts.py
https://github.com/jessecooper/pyetrade/blob/master/tests/test_accounts.py#L49

Reply:
i think i did it correctly, unit testing was never my strong suit, so please let me know if i did it wrong.

Reply:
ok fixed the unit-tests, i was able to test it all after setting up my local env, all tests pass now.